api/projects/inventory_test.go

```diff
@@ -21,7 +22,7 @@ func TestIsValidInventoryPath(t *testing.T) {
 		t.Fatal(" a path out of the cwd should be invalid")
 	}
 
-	if IsValidInventoryPath("c:\\test\\inventory") {
+	if runtime.GOOS == "windows" && IsValidInventoryPath("c:\\test\\inventory") {
```

Comment:
I'm not sure that this will ever get run, as it's impossible to test in windows on CircleCI :grimacing: 
But it's good to have as a test anyway!